File: 1160.java
int PA, PB, T, ANOS ;
        double G1, G2;

        ANOS = 0;
        T = 6;
        PA = 90000;
        PB = 120000;
        G1 = 5.5;
        G2 = 3.5;
        
        for(int i = 0; i < T; i++) {
            double PA1, PB1;
            while(PA <= PB) {
               PA1 = ((PA * G1)/100)+PA;
               PB1 = ((PB * G2)/100)+PB;

               PA = (int)PA1;
               PB = (int)PB1;

               ANOS++;
            }
            if(ANOS > 100)
             System.out.println("Mais de 1 seculo.");

            else
                System.out.println(ANOS +" anos");
        }

File: 1160.js
var input = require('fs').readFileSync('1160.txt', 'utf8');
var lines = input.split('\r\n');

var T = parseInt( lines[0]);


var PA = parseInt( lines[1]);
var PB = parseInt( lines[1]);

var ANOS = parseInt;

var G1 = parseFloat( lines[1]);
var G2 = parseFloat( lines[1]);


for(var i = 0; i < T; i++) {

    var X = parseInt( lines[i]);

    var X = PA.split(" ")+PB.split(" ")+G1.split(" ")+G2.split(" ");


    while(PA <= PB) {

       var PA1 = ((PA * G1)/100)+PA;
       var PB1 = ((PB * G2)/100)+PB;

       var PA = PA1;
       var PB = PB1;

       ANOS++;
    }
    if(ANOS > 100) {
        console.log("Mais de 1 seculo.");
    }
    
    else {
        console.log(ANOS +" anos");
    }
      
}